systemtest/src/test/java/io/strimzi/systemtest/operators/RecoveryST.java

```diff
@@ -228,6 +235,42 @@ void testRecoveryFromKafkaBridgeMetricsConfigDeletion() {
         timeMeasuringSystem.stopOperation(timeMeasuringSystem.getOperationID());
     }
 
+    @Test
+    void testRecoveryFromImpossibleMemoryRequest() {
+        String clusterName = "my-cluster";
+
+        Map<String, Quantity> requests = new HashMap<>(2);
+        requests.put("memory", new Quantity("465458732Gi"));
+
+        ResourceRequirements resourceReq = new ResourceRequirementsBuilder()
+            .withRequests(requests)
+            .build();
+
+        KafkaResource.kafkaWithoutWait(KafkaResource.defaultKafka(clusterName, 3, 3)
+            .editSpec()
+                .editKafka()
+                    .withResources(resourceReq)
+                .endKafka()
+            .endSpec()
+            .build());
+
+        PodUtils.waitForPendingPod(KafkaResources.kafkaStatefulSetName(clusterName));
```

Comment:
Shouldn't we have there some additional wait to see that pods are rly broken?

Reply:
You mean some wait for Kafka is `NotReady`? Or do you want some stability wait?

Reply:
Ye, maybe some stability wait